driver/Inc/printf.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <stdbool.h>
+#include "UART.h"
+
+extern int printf ( const char * format, ... );
```

Comment:
Do we have a correct varargs impl on our boards?

Reply:
I believe so... is it not part of C standard to have? it compiled and ran properly.

Reply:
Oh well then probably fine